src/SDKs/Resource/Management.ResourceManager/Generated/Models/CreateGroupRequest.cs

```diff
@@ -0,0 +1,64 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Management.ResourceManager.Models
+{
+    using Newtonsoft.Json;
+    using System.Linq;
+
+    /// <summary>
+    /// Management group creation parameters.
+    /// </summary>
+    public partial class CreateGroupRequest
```

Comment:
CreateGroup is too generic IMHO, people may confuse this to resource groups. Can you please make this CreateManagementGroupRequest instead?

Reply:
Our swagger is already approved and the pyhton sdk is out too. Maybe we can change this in the future releases.

Reply:
That shouldn't have been done. There are commands in CLI under az named "group", that does resource group operations. This will definitely cause confusion among our SDK users who want to create resource groups. @ravbhatnagar to comment